src/pages/guides/develop/coi.md

```diff
@@ -0,0 +1,143 @@
+
+To enable this environment yourself, perform the following steps:
+
+First, launch your browser’s developer tools and navigate to the "Network" tab, and then navigate to [Adobe Express](https://new.express.adobe.com) in the browser. Your network panel should fill up with a lot of network traffic, like this:
+
+![Network panel screenshot](./img/coi-test-1.png)
+
+To apply an override, right click on the entry for "new.express.adobe.com" and select **Override Headers**.
+
+![Override headers screenshot](./img/coi-test-2.png)
+
+Assuming you haven’t done this before, the developer tools will ask you to pick a folder on your local file system where these overrides are stored. The alert is easy to miss, since it doesn’t present as a dialog box, but rather a message near the top of your developer tool window.
+
+![Override storage folder screenshot](./img/coi-test-3.png)
+
+Click **Select Folder**. This will open your browser’s file picker. You’ll likely want to create a new folder for this step. You can put this anywhere you’d like. For example, in the following image we've selected the **Downloads** folder.
+
+![Override downloads folder screenshot](./img/coi-test-4.png)
+
+Depending on your operating system and location of the folder, the developer tools may need to request additional permissions in order to access the folder. If so, the message will again appear near the top of the developer tool window.
+
+![Allow access screenshot](./img/coi-test-5.png)
+
+Click **Allow**. This may prompt an operating system level permissions prompt. In that case, be sure to allow that as well:
+
+![OS permissions prompt screenshot](./img/coi-test-6.png)
+
+Once permissions have been granted, navigate to the **Sources** panel:
+
+![Sources panel screenshot](./img/coi-test-7.png)
+
+Next, click the **`>>`** icon and select **Overrides**:
+
+![Sources panel screenshot](./img/coi-test-8.png)
+
+Expand the entry you see there completely. You should see something like the following:
+
+![Sources panel screenshot](./img/coi-test-9.png)
+
+Click **Add override rule**:
+
+![Sources panel screenshot](./img/coi-test-10.png)
+
+Click on `header-name-1` and start entering the name of the first header -- in this case, `Cross-Origin-Embedder-Policy`.
+
+![Sources panel screenshot](./img/coi-test-11.png)
+
+Press TAB or click into the field that says "header value" and update it to the appropriate value -– in this case, `credentialless`. Once done, click the **+** icon to add the next header.
+
+![Credentialless header value](./img/coi-test-12.png)
+
+You’ll want to add headers until your overrides look like the following:
+
+![Final header values screenshot](./img/coi-test-13.png)
+
+At this point you can reload Adobe Express and proceed to test your add-on. Be sure to watch the network panel for errors that your add-on might encounter.
+
+### What to test in your add-on
+
+You should test flows in your add-on that involve the following:
```

Comment:
[Optional Suggestion to make it less wordy]Test the following flows in your add-on: